client/hotrod-client/src/test/java/org/infinispan/client/hotrod/SecureExecTest.java

```diff
@@ -156,7 +156,8 @@ private void runTestWithGivenScript(Configuration config, String scriptPath) thr
          uploadScript(scriptName, script);
       }
 
-      String result = remoteCacheManager.getCache(CACHE_NAME).execute(scriptName, params);
+      String result = remoteCacheManager.getCache(CACHE_NAME)
```

Comment:
revert?

Reply:
Yeah, not sure how this change snuck in here :)